yarn.lock

```diff
@@ -140,9 +140,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/helper-create-class-features-plugin@npm:^7.14.5":
-  version: 7.14.5
-  resolution: "@babel/helper-create-class-features-plugin@npm:7.14.5"
+"@babel/helper-create-class-features-plugin@npm:^7.14.5, @babel/helper-create-class-features-plugin@npm:^7.14.6":
```

Comment:
Rather only bump your package instead of all. Specifically stuff like babel really needs a local test first before bumping - despite patches, have had issues in the past.

So simplest is just -

- bump version in package.json
- yarn